### DISCUSSION TITLE
(mini.pick) How to search using clipboard content?

### DISCUSSION THREAD

**Discussion by mainendra** [nvim-mini/mini.nvim#665] [Q&A] [answered]
Is there a way to search using clipboard (system) content? Pasting in picker input seems not working for me.
Note: I'm using a M1 mac.

**Comment by mainendra** [comment]
<img width="899" alt="keymap" src="https://github.com/echasnovski/mini.nvim/assets/1897340/e255cb35-b3c5-43d6-8566-416d732d87fe">

Looks like `<C-R>` is registered for minibracked. I've tried mapping paste to `<C-V>`, still not working. am I missing something? 🤔

**Reply by mainendra** [reply, marked as answer]
Oh, `<C-V>` is mapped to vertical split. Issue with key mapping. Closing this ticket. Thanks.

**Reply by echasnovski** [reply]
Mappings inside 'mini.pick' are not the same as general purpose mappings (created with `vim.keymap.set()` and friends). There is [`<C-r>` inside 'mini.pick'](https://github.com/echasnovski/mini.nvim/blob/2e2931b3822bf357d2f69cbc0bcb2d99a3f0d4c8/doc/mini-pick.txt#L710) to paste from register. System register is one of `+` or `*`.

So posting from system register is supposed to be done with `<C-r>+` (or `<C-r>*`) when 'mini.pick' is active.

**Reply by mainendra** [reply]
I see, awesome. Working fine. Thanks.
